test/wallet-http-test.js

```diff
@@ -0,0 +1,1078 @@
+    assert.equal(info.chain.height, height);
+
+    const accountInfo = await wallet.getAccount('default');
+    // each coinbase output was indexed
+    assert.equal(accountInfo.balance.coin, height);
+
+    const coins = await wallet.getCoins();
+    // the wallet has no previous history besides
+    // what it has mined
+    assert.ok(coins.every(coin => coin.coinbase === true));
+  });
+
+  it('should create a transaction', async () => {
+    const tx = await wallet.createTX({
+      outputs: [{ address: cbAddress, value: 1e4 }]
+    });
+
+    assert.ok(tx);
+    assert.equal(tx.outputs.length, 1 + 1); // send + change
+    assert.equal(tx.locktime, 0);
+  });
+
+  it('should create a transaction with a locktime', async () => {
+    const locktime = 8e6;
+
+    const tx = await wallet.createTX({
+      locktime: locktime,
+      outputs: [{ address: cbAddress, value: 1e4 }]
+    });
+
+    assert.equal(tx.locktime, locktime);
+  });
+
+  it('should create a transaction that is not bip 69 sorted', async () => {
+    // create a list of outputs that descend in value
+    // bip 69 sorts in ascending order based on the value
+    const outputs = [];
+    for (let i = 0; i < 5; i++) {
+      const addr = await wallet.createAddress('default');
+      outputs.push({ address: addr.address, value: (5 - i) * 1e5 });
+    }
+
+    const tx = await wallet.createTX({
+      outputs: outputs,
+      sort: false
+    });
+
+    // assert outputs in the same order that they were sent from the client
+    for (const [i, output] of outputs.entries()) {
+      assert.equal(tx.outputs[i].value, output.value);
+      assert.equal(tx.outputs[i].address.toString(network), output.address);
+    }
+
+    const mtx = MTX.fromJSON(tx);
+    mtx.sortMembers();
+
+    // the order changes after sorting
+    assert.ok(tx.outputs[0].value !== mtx.outputs[0].value);
+  });
+
+  it('should create a transaction that is bip 69 sorted', async () => {
+    const outputs = [];
+    for (let i = 0; i < 5; i++) {
+      const addr = await wallet.createAddress('default');
+      outputs.push({ address: addr.address, value: (5 - i) * 1e5 });
+    }
+
+    const tx = await wallet.createTX({
+      outputs: outputs
+    });
+
+    const mtx = MTX.fromJSON(tx);
+    mtx.sortMembers();
+
+    // assert the ordering of the outputs is the
+    // same after sorting the response client side
+    for (const [i, output] of tx.outputs.entries()) {
+      assert.equal(output.value, mtx.outputs[i].value);
+      assert.equal(output.address, mtx.outputs[i].address.toString(network));
+    }
+  });
+
+  it('should mine to the secondary/default wallet', async () => {
+    const height = 5;
+
+    const {address} = await wallet2.createAddress('default');
+    await mineBlocks(height, address);
+
+    const accountInfo = await wallet2.getAccount('default');
+    assert.equal(accountInfo.balance.coin, height);
+  });
+
+  it('should have no name state indexed', async () => {
```

Comment:
nit: could we add "initially" to the end of this, i.e. 'should have no name state indexed initially'.